canary/webrtc-c/src/Peer.cpp

```diff
@@ -602,6 +603,13 @@ STATUS Peer::writeFrame(PFrame pFrame, MEDIA_STREAM_TRACK_KIND kind)
             retStatus = STATUS_SUCCESS;
         }
 
+        else if (STATUS_SUCCEEDED(retStatus) && this->firstFrame && this->isMaster) {
+            this->firstFrame = FALSE;
+            this->startupLatency = DOUBLE(GETTIME() - this->offerReceiveTimestamp) / HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
```

Comment:
This is a bad construct. Perhaps what you meant is this?

this->startupLatency = (DOUBLE) (GETTIME() - this->offerReceiveTimestamp) / HUNDREDS_OF_NANOS_IN_A_MILLISECOND;

Reply:
Uh-oh! Typo :(